src/main/java/com/at/avro/DbSchemaExtractor.java

```diff
@@ -101,17 +93,17 @@ public AvroSchema getForTable(AvroConfig avroConfig, String dbSchemaName, String
             throw new IllegalArgumentException("Can not get connection to " + connectionUrl, e);
         }
         catch (SchemaCrawlerException e) {
-            throw new RuntimeException(e);
```

Comment:
I would prefer to throw an exception here. Exception lets the caller know that there's an error and gives an ability to see what's wrong. The empty list hides an issue while not providing any useful data.

Reply:
this change was appied **only** to support **current test cases**. 
if you'd like to see other behaviour - let's create a new issue and then create a new PR.